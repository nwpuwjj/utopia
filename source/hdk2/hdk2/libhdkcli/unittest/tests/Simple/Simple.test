#
# Simple.test
#   Test the basic functionality of libhdkcli.
#

method: http://cisco.com/Unittest/SimpleUnittestMethod
host: http://127.0.0.1:8080
username: admin
password: password
timeout: 8
network-object-id: 00000001-0020-0300-4000-908030040000