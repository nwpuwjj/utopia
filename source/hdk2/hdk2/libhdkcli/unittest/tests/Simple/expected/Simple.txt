+++++++ Request Headers +++++++
HTTP/1.1 POST /HNAP1
Connection: Close
Content-Type: text/xml; charset="utf-8"
Content-Length: 316
Authorization: Basic YWRtaW46cGFzc3dvcmQ=
SOAPAction: "http://cisco.com/Unittest/SimpleUnittestMethod"
Host: 127.0.0.1:8080
X-NetworkObjectID: 00000001-0020-0300-4000-908030040000

+++++++ Request Data +++++++
<?xml version="1.0" encoding="utf-8"?>
<Envelope xmlns="http://schemas.xmlsoap.org/soap/envelope/">
<Body>
<SimpleUnittestMethod xmlns="http://cisco.com/Unittest/">
<StringArg0>a string argument &amp; 1 &lt;</StringArg0>
<BoolArg0>true</BoolArg0>
<IntArg0>-1000</IntArg0>
</SimpleUnittestMethod>
</Body>
</Envelope>


------- Response -------
HTTP/1.1 200 OK
Content-Type: text/xml; charset=utf-8
Connection: close
Content-Length: 321
$sleep: 1000.0

------- Response -------

<?xml version="1.0" encoding="utf-8" ?>
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
<soap:Body>
  <SimpleUnittestMethodResponse xmlns="http://cisco.com/Unittest/">
      <SimpleUnittestMethodResult>OK</SimpleUnittestMethodResult>
  </SimpleUnittestMethodResponse>
</soap:Body>
$sleep: 1000.0

------- Response -------
</soap:Envelope>

******* Client Result ******

      SOAP action: http://cisco.com/Unittest/SimpleUnittestMethod
        HTTP host: http://127.0.0.1:8080
Network Object ID: 00000001-0020-0300-4000-908030040000
    HTTP username: admin
    HTTP password: password
          timeout: 8
       HNAP input: Simple.input.xml

<?xml version="1.0" encoding="utf-8"?>
<Envelope xmlns="http://schemas.xmlsoap.org/soap/envelope/">
<Body>
<SimpleUnittestMethodResponse xmlns="http://cisco.com/Unittest/">
<SimpleUnittestMethodResult>OK</SimpleUnittestMethodResult>
</SimpleUnittestMethodResponse>
</Body>
</Envelope>
malloc_interposer.c - ************************************************************
malloc_interposer.c - exit: malloc called 80 times
malloc_interposer.c - exit: free called 80 times
malloc_interposer.c - exit: malloc current = 0
malloc_interposer.c - exit: malloc total = 18740
malloc_interposer.c - exit: malloc max = 11168
malloc_interposer.c - ************************************************************
